remoteit/desktop
# [DESK-768] Linked accounts menu update

### Description


Account linking menus refactoring, CLI v1.6.8
### Test plan



1. Sign in
2. Go to settings page
3. You should see only one linked account ment
4. Go to device list and in an account that has linked you to other accounts, you should see the device list switcher menu. 
5. You should be able to leave a linked account from an icon in the switcher menu

### Ticket



<https://remoteit.atlassian.net/browse/DESK-768>
